includes/library/zencart/Paginator/src/adapters/ArrayFactory.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Class ArrayFactory
+ *
+ * @copyright Copyright 2003-2016 Zen Cart Development Team
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id: $
+ */
+namespace ZenCart\Paginator\adapters;
+
+use ZenCart\Paginator\AdapterInterface;
+use ZenCart\Paginator\AbstractAdapter;
+
+/**
+ * Class ArrayFactory
+ * @package ZenCart\Paginator\adapters
+ */
+class ArrayFactory extends AbstractAdapter implements AdapterInterface
```

Comment:
Note the class isn't a Factory, it's my fault for calling the other adapter QueryFactory.
I called it that as it is operating on a QueryFactory($db) sql query.